pkgs/by-name/ch/chirpstack-udp-forwarder/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  darwin,
+  rustPlatform,
+  fetchFromGitHub,
+  gitUpdater,
+  protobuf,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "chirpstack-udp-forwarder";
+  version = "4.1.8";
+
+  src = fetchFromGitHub {
+    owner = "chirpstack";
+    repo = "chirpstack-udp-forwarder";
+    rev = "v${version}";
+    hash = "sha256-Snj5nKyFsq8WJJNw1d8O/YX/dZ/tCTVBw5R8kXJvsa4=";
+  };
+
+  cargoHash = "sha256-7ugrIVT4SYrqPqF0CrLU+/Ep/p9H7/on3hkZ5JzY9AE=";
+
+  nativeBuildInputs = [ protobuf ];
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk.frameworks.CoreFoundation
+    darwin.apple_sdk.frameworks.Security
+  ];
+
```

Comment:
Would be better to add https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook

Reply:
Fixed in 46d1979f0fa6